### PR TITLE
Announcing Scala.js 1.0.0-M5.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

File: doc/internals/downloads.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ We strongly recommend using the SBT plugin, as shown in the [bootstrapping skele
 
 The CLI distribution requires `scala` and `scalac` (of the right major version) to be on the execution path. Unpack it wherever you like and add the `bin/` folder to your execution path.
 
+#### Scala.js 1.0.0-M4
+* [1.0.0-M4, Scala 2.12 (tgz, 16MB)]({{ site.production_url }}/files/scalajs_2.12-1.0.0-M4.tgz)
+* [1.0.0-M4, Scala 2.12 (zip, 16MB)]({{ site.production_url }}/files/scalajs_2.12-1.0.0-M4.zip)
+* [1.0.0-M4, Scala 2.11 (tgz, 28MB)]({{ site.production_url }}/files/scalajs_2.11-1.0.0-M4.tgz)
+* [1.0.0-M4, Scala 2.11 (zip, 28MB)]({{ site.production_url }}/files/scalajs_2.11-1.0.0-M4.zip)
```
Comment (sjrd) [inline doc/internals/downloads.md]: The file sizes here will need to be adapted when the CLI is built.

**Comment by sjrd** [comment]
Updated for 1.0.0-M5.

File: doc/all-api.md [hunk 2]
```diff
@@ -5,6 +5,24 @@ title: All previous versions of the Scala.js API
 
 ## All previous versions of the API
 
+### Scala.js 1.0.0-M4
```
Comment (gzm0) [inline doc/all-api.md]: Needs to be changed to M5

Comment (sjrd) [reply doc/all-api.md]: Oops, indeed. Fixed.

**Comment by sjrd** [comment]
Rebased and adjusted for announcing tomorrow (August 13).

**Pull request merged**